site/src/components/progress-bar.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Progress Bar
+description: The progress-bar component in the Atlas Design System
+template: standard
+figmaEmbed:
+classType: Component
+classPrefixes:
+  - progress-bar
+---
+
+# Progress Bar
+
+Use the progress-bar component to communicate how much of a task has been completed.
+
+## Usage
+
+Here is an example of a standard usage of the progress bar component.
```

Comment:
CSS-only progressbars will cause accessibility issues, since screenreaders and other assistive tech won't be able to make sense of them.

Can we include some guidance about using [the `<progress>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress) with this Atlas component, or about adding the necessary ARIA roles?

I'd be happy to chip in for that guidance if need be.